akka-projection-slick/src/main/scala/akka/projection/slick/internal/SlickProjectionImpl.scala

```diff
@@ -150,8 +153,10 @@ private[projection] class SlickProjectionImpl[Offset, Envelope, P <: JdbcProfile
             .mapAsync(parallelism = 1)(storeOffset)
       }
 
-    Source
-      .futureSource(futSource)
+    RestartSource
+      .withBackoff(projectionConfig.minBackoff, projectionConfig.maxBackoff, projectionConfig.randomBackoffFactor) {
+        () => Source.futureSource(futSource)
+      }
```

Comment:
this is only for the readOffsets? shouldn't it be around everything so that it covers `handlerFlow` also?

Reply:
yes, of course.

Reply:
hmm, now I remember why this happened. I got a failure because the kills switch needs to be on the `RestartSource`

Reply:
same mistake is done in the Cassandra impl